src/main.cpp

```diff
@@ -1,15 +1,26 @@
+/**
+ * @file
+ * @brief     Main function
+ * @author    Jeroen van Hattem
+ * @license   MIT
+ */
 #include "robot_arm.hpp"
+#include "settings.hpp"
 #include "wrap-hwlib.hpp"
 
 int main() {
     WDT->WDT_MR = WDT_MR_WDDIS;
     int coordinates[3] = {4, 12, 5};
+    int speed = 5000;
 
     hwlib::wait_ms(1000);
 
-    robot_arm uarm_swift_pro(hwlib::target::pin_out(hwlib::target::pins::d7));
+    RobotArm uarmSwiftPro;
+    uarmSwiftPro.move(coordinates, speed);
 
-    uarm_swift_pro.move(coordinates);
+    hwlib::cout << uarmSwiftPro.getCoordinates('x') << hwlib::endl;
+    hwlib::cout << uarmSwiftPro.getCoordinates('y') << hwlib::endl;
+    hwlib::cout << uarmSwiftPro.getCoordinates('z') << hwlib::endl;
 
```

Comment:
i like to see the main as clean as posible because it's about the code in the classes

Reply:
Should be removed and not commented.